mod-orders-storage/schemas/physical.json

```diff
@@ -3,6 +3,17 @@
   "description": "purchase order line physical material details",
   "type": "object",
   "properties": {
+    "createInventory": {
+      "description": "Shows what inventory objects need to be created for physical resource",
+      "type": "string",
+      "enum": [
+        "INSTANCE_HOLDING_ITEM",
+        "INSTANCE_HOLDING",
+        "INSTANCE",
+        "NONE"
+      ],
+      "default": "INSTANCE_HOLDING_ITEM"
```

Comment:
Same as for e-resource - remove default. As described in [MODORDERS-178](https://issues.folio.org/browse/MODORDERS-178):
> NOTE: The defaults identified above should NOT be set in the schemas, but rather in the code. There are a few reasons for this... We plan on later implementing the ability for tenants to specify defaults which override the system-wide defaults noted in this story. Another reason relates to orders of type "Other" and is described below.